presto-hive/src/main/java/com/facebook/presto/hive/util/HudiRealtimeSplitConverter.java

```diff
@@ -73,6 +73,7 @@ public Optional<FileSplit> recreateFileSplitWithCustomInfo(FileSplit split, Map<
                     requireNonNull(customSplitInfo.get(HUDI_BASEPATH_KEY), "HUDI_BASEPATH_KEY is missing"),
                     deltaLogFiles,
                     requireNonNull(customSplitInfo.get(HUDI_MAX_COMMIT_TIME_KEY), "HUDI_MAX_COMMIT_TIME_KEY is missing"),
+                    false,
```

Comment:
Can we add a comment here as to why this is `false`? Essentially, when we add incremental query support then we will have to make it configurable. For now, we can simpy let the readers be aware about this.

Reply:
just fyi, it is false because we don't yet support incremental query for hudi table through presto.

Reply:
Fine. Added a comment line here.